pkg/controller/secret/secret_controller.go

```diff
@@ -278,7 +278,7 @@ func createWatchdogRoute() *alertmanager.Route {
 	return &alertmanager.Route{
 		Receiver:       receiverWatchdog,
 		RepeatInterval: "5m",
-		Match:          map[string]string{"alertname": "Watchdog"},
+		Match:          map[string]string{"alertname": "Watchdog", "namespace": "openshift-monitoring"},
```

Comment:
I don't think this is needed. This will break versions that don't have this label. Matching on the alertname is good enough.

Reply:
ah good point. I suggested this so blame me :)